integration/chunks_storage_backends_test.go

```diff
@@ -24,27 +24,44 @@ func TestChunksStorageAllIndexBackends(t *testing.T) {
 	dynamo := e2edb.NewDynamoDB()
 	bigtable := e2edb.NewBigtable()
 	cassandra := e2edb.NewCassandra()
-
-	stores := []string{"aws-dynamo", "bigtable", "cassandra"}
+	elasticsearch := e2edb.NewElasticSearch()
+
+	// index store : chunk store
+	stores := map[string]string{
```

Comment:
Maps do not guarantee ordering. Please switch back to a slice using a custom struct, ie.

```
stores := []struct{
   IndexStore string
   ChunkStore string
}{
  // ...
}
```

Reply:
though I don't see any ordering problem here, but fair enough, I will change to struct array.